src/vdaf/prio3.rs

```diff
@@ -143,13 +144,27 @@ fn check_num_aggregators(num_aggregators: u8) -> Result<(), VdafError> {
 }
 
 /// The base type for prio3.
+#[derive(Derivative)]
+#[derivative(Debug)]
```

Comment:
It seems like what this line is doing is allowing us to ignore `phantom` for the purposes of deriving the `Debug` trait. Why is this necessary here, but not for hits?

Reply:
In the hits case, we have `PhantomData<I>`, where `I: Idpf<2, 2>`, and trait `Idpf` requires `Debug`. The `A` type parameter on `Prio3` has no trait bound at all, not even Debug. I have to admit, I don't fully understand this, because [`PhantomData<T>` should be Debug for all `T`](https://doc.rust-lang.org/std/marker/struct.PhantomData.html#impl-Debug), but if I don't do this, the compiler is mad.

Reply:
What about
```rust
pub struct Prio3<T: Type, A: Debug> {
   ...
}
```
In any case, we could solve this by taking https://github.com/abetterinternet/libprio-rs/pull/135#discussion_r746094366